src/MainWindow.ui

```diff
@@ -1288,6 +1290,24 @@
     <string>Export as &amp;AMF...</string>
    </property>
   </action>
+  <action name="fileActionExportX3D">
+   <property name="icon">
+    <iconset resource="../openscad.qrc">
+     <normaloff>:/images/X3D.png</normaloff>:/images/X3D.png</iconset>
```

Comment:
Do you have a way to generate new icons, a script? Where do the existing come from?